cosi.go

```diff
@@ -1,5 +1,7 @@
-// Cosi takes a file or a message and signs it collectively.
-// For usage, see README.md
+// CoSi is a scalable protocol for collectively signing messages.
+// CoSi produces compact signatures that clients can verify efficiently,
+// and that convey the precise set of cosigners for transparency.
+// For details see https://github.com/dedis/cosi/blob/master/README.md.
```

Comment:
I'd redirect simply to the repository  main page, namely https://github.com/dedis/cosi.